lib/pennyworth/commands/setup_command.rb

```diff
@@ -62,6 +62,20 @@ def read_os_release_file
       File.read(os_release_file) if File.exist?(os_release_file)
     end
 
+    def vagrant_installed?
+      @vagrant = Cheetah.run "rpm", "-q", "vagrant", stdout: :capture
+      @vagrant_version = @vagrant.lines.select{|plugin| plugin.start_with?("vagrant")}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.